websites/P/Peloton/dist/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+    "author": {
+        "name": "The Gamerzs",
+        "id": "712294190339588209"
+    },
+    "url": [
+        "www.onepeloton.com"
+    ],
+    "description": {
+        "en": "Peloton is attractive, addictive, and seriously whips you into shape."
+    },
+    "regExp": "([a-z0-9-]+[.])*onepeloton([.][a-z]|[co.uk])",
```

Comment:
![image](https://user-images.githubusercontent.com/20159369/82520035-c5cf2a80-9ae8-11ea-9182-7cbc21a3c2d6.png)
I think you may need to readjust your regex, unless this is correct behavior.

Reply:
TLD change between countries but have the same file names and file structure

Reply:
Subdomains are too many to specify

Reply:
`([a-z0-9-]+[.])*onepeleton([.][a-z]+)+[/]`

Reply:
![image](https://user-images.githubusercontent.com/34792838/82560107-0ecac180-9b71-11ea-842e-ecab7ba52614.png)